ci/test/02_run_container.py

```diff
@@ -45,6 +58,30 @@ def main():
                 file.write(f"{k}={v}\n")
     run(["cat", env_file])
 
+    if not os.getenv("DANGER_RUN_CI_ON_HOST"):
+        CI_IMAGE_LABEL = "bitcoin-ci-test"
+
+        maybe_cache_arg = shlex.split(os.getenv("DOCKER_BUILD_CACHE_ARG", ""))
```

Comment:
Is this related to the mentioned https://www.shellcheck.net/wiki/SC2086? Because we need split to handle quotes and escaping properly?